index.js

```diff
@@ -16,6 +16,7 @@ var PinkiePromise = require('pinkie-promise');
 var unzipResponse = require('unzip-response');
 var createErrorClass = require('create-error-class');
 var nodeStatusCodes = require('node-status-codes');
+var isPlainObject = require('is-plain-object');
```

Comment:
Use https://github.com/sindresorhus/is-plain-obj instead. I made it specifically because `is-plain-object` were suboptimal.